pkg/kubelet/kubelet_getters.go

```diff
@@ -240,13 +240,18 @@ func (kl *Kubelet) GetNode() (*v1.Node, error) {
 // Return kubelet's nodeInfo for this node, except on error or if in standalone mode,
 // in which case return a manufactured nodeInfo representing a node with no pods,
 // zero capacity, and the default labels.
-func (kl *Kubelet) getNodeAnyWay() (*v1.Node, error) {
+func (kl *Kubelet) getNodeAnyWay() (node *v1.Node, err error) {
```

Comment:
Would you mind sharing more about why we are now using named return values? From my reading, I'm not sure its necessary and I believe it makes the code slightly less readable.